src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_driver_distraction_notification.cc

```diff
@@ -126,6 +127,21 @@ void OnDriverDistractionNotification::Run() {
       static_cast<int32_t>(application_manager::MessageType::kNotification);
   (*on_driver_distraction)[strings::msg_params][mobile_notification::state] =
       state;
+  if ((*message_)[strings::msg_params].keyExists(
+          mobile_notification::lock_screen_dismissal_enabled)) {
+    const bool is_lock_screen_dismissal_enabled =
+        (*message_)[strings::msg_params]
+                   [mobile_notification::lock_screen_dismissal_enabled]
+                       .asBool();
+    application_manager_.set_lock_screen_dismissal_state(
+        is_lock_screen_dismissal_enabled);
+    (*on_driver_distraction)
+        [strings::msg_params]
+        [mobile_notification::lock_screen_dismissal_enabled] =
+            is_lock_screen_dismissal_enabled;
+  } else {
+    application_manager_.set_lock_screen_dismissal_state(-1);
```

Comment:
@IGapchuk what should be assign in 
`(*on_driver_distraction)
        [strings::msg_params]
        [mobile_notification::lock_screen_dismissal_enabled]`
in else case ?

Reply:
@ZhdanovP Nothing, because above exists check for existing key in message and if this key not exists in message, we could can't set any value to message.